recipes/dj-static/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set package = "dj-static" %}
```

Comment:
I think I'll eat my words now :yum:. `conda build` works just fine with variables... I swear tried that before writing this overly complicated `sed` script.

@pelson Does this have to be resolved as a "yes or no"?  I think we should allow some freedoms to the individual package maintainers to construct their `meta.yaml` in a way that allows them to keep up the maintenence. Adding additional variables to the top might suit me or someone else who constructs their `yaml` programmatically and sorts the top level keys so `package` won't necessarily be at the top (thats the best example I could come up with).